core/requirements.txt

```diff
@@ -0,0 +1,3 @@
+click==6.7
+-e git+https://github.com/SecurityInnovation/PGPy.git@release/0.4.1#egg=PGPy-0.4.1
```

Comment:
please use a release from pypi, not some git reference -- also what do you need "requirements.txt" for?